babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -788,15 +788,70 @@ SqlIndex SqlCreateTableIndex(Span s) :
 SqlNode SqlExecMacro() :
 {
     SqlIdentifier macro;
+    SqlNodeList paramNames = new SqlNodeList(getPos());
+    SqlNodeList paramValues = new SqlNodeList(getPos());
     Span s;
 }
 {
     macro = CompoundIdentifier() { s = span(); }
+    [SqlExecMacroArgument(paramNames, paramValues)]
```

Comment:
nit: space between "[" brackets and statement

Reply:
Put in different line in https://github.com/googleinterns/calcite/pull/108/commits/7896e626fa4d16a2a8ac9b734f38f00f5cbfb6d2